model/experiment_list.py

```diff
@@ -743,6 +743,8 @@ def from_serialized_format(filename, check_format=True):
         # First try as a JSON file
         try:
             return ExperimentListFactory.from_json_file(filename, check_format)
+        except FileNotFoundError:
```

Comment:
```suggestion
        except OSError:
```

Reply:
It looks like it's actually `IOError` not `OSError`, at least the test now passes for me on both Python 2 and Python 3.